test/integration/fleetscope/fleetscope_test.go

```diff
@@ -163,6 +163,27 @@ func TestFleetscope(t *testing.T) {
 					assert.Equal(gkeScopes, opGKEScopes.Get("name").String(), fmt.Sprintf("The GKE Namespace should be %s", gkeScopes))
 					assert.True(opGKEScopes.Exists(), "Namespace %s should exist", gkeScopes)
 				}
+				gkeMeshCommand := fmt.Sprintf("beta container fleet mesh describe --project %s", clusterProjectID)
+				pollMeshProvisioning := func(cmd string) func() (bool, error) {
+					return func() (bool, error) {
+						retry := false
+						result := gcloud.Runf(t, cmd).Array()
+						if len(result) < 1 {
+							return true, nil
+						}
+						for _, memberShipName := range membershipNames {
+							dataPlaneManagement := result[0].Get(fmt.Sprintf("membershipStates.%s.servicemesh.dataPlaneManagement.state", memberShipName)).String()
```

Comment:
I suspect this isn't working as intended as this key uses project number instead of project ID

For example: "projects/938647760329/locations/us-central1/memberships/cluster-us-central1-development"